app/helpers/sessions_helper.rb

```diff
@@ -21,6 +21,10 @@ def current_user
     end
   end
 
+  def current_user?(user)
+    user && user == current_user
```

Comment:
object thì chỉ nên so sánh 1 attribute nào đó, ko cần ss hết như thế này
```suggestion
    user && user.id == current_user.id
```